boudicca.base/common-model/src/commonMain/kotlin/base/boudicca/format/URI.kt

```diff
@@ -0,0 +1,7 @@
+package base.boudicca.format
+
+
+expect class URI
+
+expect fun parseURI(uri: String): URI
```

Comment:
can we somehow group these parsing and formatting functions? right now they are floating around as global functions.

Reply:
i think we can group it as an object, i will try